Makefile

```diff
@@ -54,6 +54,8 @@ ifeq ($(OS),Windows_NT)
     CGO_LDFLAGS += -static -static-libgcc -static-libstdc++
     CGO_LDFLAGS_EXTRA += -lversion
 
+	BINARYEN_OPTION += -DCMAKE_EXE_LINKER_FLAGS='-static-libgcc -static-libstdc++'
```

Comment:
I'm almost embarrassed to point out something so trivial as an extra tab here :joy_cat:

Reply:
Huh. I guess this is what happens when using 4-wide tabs in editor :facepalm: